### ISSUE TITLE
Review Page is not working in Pixelwise(beta) mode

### ISSUE THREAD

**Issue by daniel-code** [Labelbox/Labelbox#132]
After multi collaborators label pixelwise feature, it display same label in same image at review page and export same mask. 
In vector node, it display different collaborators's labels in same image at review page and export each mask of collaborators in same image.

**Comment by manuaero** [comment]
Thanks -- we will be fixing this soon

**Comment by euwest** [comment]
@daniel-code are you still experiencing this issue?

**Comment by daniel-code** [comment]
Yes, it still not work well. In review page, it still display same label in same image at review page and export same mask.
